packages/app-project/src/screens/ClassifyPage/components/YourStats/YourStats.spec.js

```diff
@@ -1,15 +1,41 @@
-import { shallow } from 'enzyme'
+import { render, screen } from '@testing-library/react'
+import { composeStory } from '@storybook/react'
+import Meta, { YourStats } from './YourStats.stories.js'
+import { YourStatsStoreMock } from './YourStats.mock'
 
-import YourStats from './YourStats'
+describe('Component > YourStats', function () {
+  beforeEach(function () {
+		const YourStatsStory = composeStory(YourStats, Meta)
+		render(<YourStatsStory />)
+	})
 
-let wrapper
+	it('should have a heading', function () {
+    expect(screen.findByText(`Your ${YourStatsStoreMock.project.display_name} Statistics`)).to.be.ok()
+  })
 
-describe('Component > YourStats', function () {
-  before(function () {
-    wrapper = shallow(<YourStats />)
+	it('should have today\'s classification count ', function () {
+    expect(screen.getAllByText(`${YourStatsStoreMock.user.personalization.counts.today}`).length).to.equal(2)
```

Comment:
See Note 2. Whatever solution is implemented to check for "Classifications Total" should also apply to "Classifications Today"